dashboards/SOC Management Dashboard.json

```diff
@@ -0,0 +1,40 @@
+{
+    "@context": "\/api\/3\/contexts\/Dashboard",
+    "@type": "Dashboard",
+    "uuid": "57ffafc7-bd34-4b0f-b126-2c91fc2b0860",
+    "displayName": "SOC Management Dashboard",
+    "type": "rows",
+    "templateType": "dashboard",
+    "roles": [],
```

Comment:
Add Roles to the Dashboard

Reply:
Added roles, Full app permission, SOC analyst, SOC manager !